src/Appium.Net/Appium/Interfaces/IInteractsWithApps.cs

```diff
@@ -56,6 +56,21 @@ public interface IInteractsWithApps : IExecuteMethod
         /// <param name="appPath">a string containing the id of the app.</param>
         void RemoveApp(string appId);
 
+        /// <summary>
+        /// Terminates an App.
+        /// </summary>
+        /// <param name="appId">a string containing the id of the app.</param>
+        /// <return>a boolean indicating if the app was terminated.</return>
+        bool TerminateApp(string appId);
+
+        /// <summary>
+        /// Terminates an App.
+        /// </summary>
+        /// <param name="appId">a string containing the id of the app.</param>
+        /// <param name="timeout">an integer number containing the timeout in ms for how long to wait until the application is terminated.</param>
+        /// <return>a boolean indicating if the app was terminated in the given timeout.</return>
+        bool TerminateApp(string appId, int timeout);
```

Comment:
Does C# have a special type for Duration, like Java does? I would prefer to use it over the generic int

Reply:
Yes, TimeSpan. I haven't used it because the timeout for BackgroundApp is an int, although here it is in seconds and -1 deactivates the app. The Timeout for the TouchAction "wait" is in ms as long (in the java client both use a duration).

So I'm not sure what to use here, I think I'm leaning towards TimeSpan...